record-and-playback/core/lib/recordandplayback/generators/events.rb

```diff
@@ -605,6 +615,72 @@ def self.get_chat_events(events, start_time, end_time, bbb_props = {})
       chats
     end
 
+    def self.create_moderators_period(events, initial_timestamp)
+      list_moderators_period = []
+
+      events.xpath('/recording/event').each do |event|
+        # BigBlueButton.logger.info("Teste aqui com: " + event.inspect)
```

Comment:
Please don't leave commented code in the final PR

Reply:
@kepstin, Ohh, sorry about that, I thought I removed all my comments, didn't see that one.